src/gui/optionsdlg.ui

```diff
@@ -90,7 +90,7 @@
      </widget>
      <widget class="QStackedWidget" name="tabOption">
       <property name="currentIndex">
-       <number>0</number>
+       <number>5</number>
```

Comment:
You have many unrelated changes in .ui files (probably created by QtCreator). Please undo it.